docker-images/postgres-12.6-alpine/rootfs/postgres.sh

```diff
@@ -20,4 +22,9 @@ fi
 
 /conf.sh
 
+if [ ! -s "${REINDEX_COMPLETED_FILE}" ]; then
+  echo "[INFO] Re-creating all indexes from for database '$POSTGRES_DB'"
```

Comment:
_from_ where? Maybe I'm reading this line wrong.

Reply:
This is a typo, my bad.
```suggestion
  echo "[INFO] Re-creating all indexes for database '$POSTGRES_DB'"
```